src/content/community/releases/_index.en.md

```diff
@@ -5,6 +5,15 @@ weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
 
+## v0.14.7 (October 17, 2023)
+
+* Fixed an issue with iptables, where xtables.lock will be created as directory and subsequent invocations of iptables on the host
+  will fail.
+* IPsec connection will be now retried if there are any `whack` errors.
+* The Pluto process now waits for 1 minute before returning the error code `33`.
+* IP forwarding will be now enabled in Submariner interfaces if not enabled by default.
+* Submariner gateway now starts pluto process only when `libreswan` connects to first remote endpoint
+
```

Comment:
Also need https://github.com/submariner-io/submariner/pull/2531:

"_The Submariner gateway now retries reading local node information on startup to reduce pod restarts if the Kubernetes API server is temporarily unavailable._"

and https://github.com/submariner-io/submariner/pull/2597:

"_Reduced data path downtime with Libreswan cable driver when gateway pod restarts._"

Reply:
The first one above was not release-noted in other branches so I assume was deemed as not needing it.
The others are already included in the release-0.14 branch. We just need the ones a noted above in the  release-0.14 branch: https://github.com/submariner-io/submariner-website/pull/1073